datamonster_api/tests/lib/regression.py

```diff
@@ -143,24 +223,22 @@ def test_data_source_2():
     assert_frame_equal(df.head(1), pandas.DataFrame.from_records([records]))
 
 
-def test_bigger_data_source():
-    """ Test SimilarWeb which is a bigger dataset
+def test_get_data_bigger():
+    """ Test `SimilarWeb Direct Volume` which is a bigger dataset
```

Comment:
Is this not static data? If it is, seems like we should add some stronger assertions on the df contents.

Reply:
the point here was just to check that we can pull in a larger data set - the stronger assertions are done on the QA ETL and factset

Reply:
kk